src/services/api.ts

```diff
@@ -377,7 +378,20 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
             path_id: tenantId?.PathId,
         });
     }
+    getChartData({target, from, until, maxDataPoints}: JsonRenderRequestParams) {
+        const requestString = `${target}&from=${from}&until=${until}&maxDataPoints=${maxDataPoints}&format=json`;
 
+        return this.post<JsonRenderResponse>(
+            this.getPath('/viewer/json/render'),
+            requestString,
+            {},
+            {
+                headers: {
+                    'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
`viewer/json/render` requires different content type compared to other api calls